eclipsePlugin/src/de/tobject/findbugs/util/DebugPrinter.java

```diff
@@ -0,0 +1,34 @@
+package de.tobject.findbugs.util;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.PrintStream;
+import java.io.UnsupportedEncodingException;
+
+public class DebugPrinter {
+	private static PrintStream ps = null;
+    private static final String outFileName = "SpotbugsDebugEclipse.txt";
+
+	private static void initMyPrinter() {
+		try {
+			FileOutputStream fs = new FileOutputStream(new File(outFileName));
+			ps = new PrintStream(fs, false, "UTF-8");
+			ps.println("Printer begion work.");
+		} catch (FileNotFoundException | UnsupportedEncodingException e) {
+			e.printStackTrace();
+		}
+	}
+
+	public static void printStrToFile(Object obj) {
+		if (ps == null) {
+			initMyPrinter();
+		}
+		String str = "null";
```

Comment:
It would be simpler to just call `String.valueOf(obj)`, which will automatically convert `null` to "null".